cmd/kwild/server/server.go

```diff
@@ -115,18 +116,20 @@ func New(ctx context.Context, cfg *config.KwildConfig, genesisCfg *config.Genesi
 
 func (s *Server) Start(ctx context.Context) error {
 	defer func() {
+		s.log.Info("Closing server resources...")
 		err := s.closers.closeAll()
 		if err != nil {
 			s.log.Error("failed to close resource:", zap.Error(err))
 		}
+		s.log.Info("Server is now shut down.")
```

Comment:
I didn't mean to sneak this in, but it would be helpful in cases like today with fractal.